parcel/ParcelModel.jl

```diff
@@ -112,17 +123,18 @@ function parcel_model(dY, Y, p, t)
 
     # number concentration and ...
     dNᵢ_dt = dNᵢ_dt_dep + dNᵢ_dt_imm + dNᵢ_dt_hom
-    dNₐ_dt = -dNᵢ_dt_dep
-    dNₗ_dt = -dNᵢ_dt_imm - dNᵢ_dt_hom
+    dNₐ_dt = -dNᵢ_dt_dep - dNₗ_dt_act
+    dNₗ_dt = dNₗ_dt_act - dNᵢ_dt_imm - dNᵢ_dt_hom
     # ... water mass budget
-    dqₗ_dt_v2l = dqₗ_dt_ce
+    dqₗ_dt_a2l = dqₗ_dt_act
+    dqₗ_dt_v2l = dqₗ_dt_ce + dqₗ_dt_act
     dqᵢ_dt_l2i = dqᵢ_dt_imm + dqᵢ_dt_hom
     dqᵢ_dt_v2i = dqᵢ_dt_dep + dqᵢ_dt_ds
 
     # Update the tendecies
     dqᵢ_dt = dqᵢ_dt_v2i + dqᵢ_dt_l2i
-    dqₗ_dt = dqₗ_dt_v2l - dqᵢ_dt_l2i
-    dqᵥ_dt = -dqᵢ_dt - dqₗ_dt
+    dqₗ_dt = dqₗ_dt_v2l - dqᵢ_dt_l2i + dqₗ_dt_a2l
```

Comment:
I think you are now counting `dqₗ_dt_a2l` twice? You already added it to `dqₗ_dt_v2l`?